src/bundle/Resources/config/graphql/ContentType.types.yaml

```
@@ -107,7 +107,7 @@ ContentTypeGroup:
                 resolve: "@=resolver('UserById', [value.modifierId])"
             contentTypes:
                 type: "[ContentType]"
-                resolve: "@=resolver('ContentTypesFromGroup', {'groupId': value.id})"
+                resolve: "@=resolver('ContentTypesFromGroup', [{'groupId': value.id}])"
```

To reduce technical debt, we try to gradually move away from the [deprecated `resolver`](https://github.com/overblog/GraphQLBundle/blob/0.14/UPGRADE.md#replace-resolver-expression-function):
```suggestion
                resolve: "@=query('ContentTypesFromGroup', [{'groupId': value.id}])"
```

---

Changed, it is not a drop in replacement tho for future reference. 
`"@=query('ContentTypesFromGroup', {'groupId': value.id})"`

---

Correct, thanks.